packages/perseus/src/widgets/interactive-graphs/graphs/components/use-control-point.tsx

```diff
@@ -53,7 +53,6 @@ export function useControlPoint(params: Params): Return {
         ariaDescribedBy,
         ariaLabel,
         ariaLive = "polite",
-        color,
```

Comment:
Removed this because the prop is not being used anywhere. All interactive colors are coming from the graph config now.